File: cap13/python/funciones1.py
from math import sqrt

def cuadrado(n):
    return n * n

def hipotenusa(a, b):
    return sqrt(cuadrado(a) + cuadrado(b))

hipotenusa = hipotenusa(3, 4)
print("La hipotenusa de un triángulo de base 3 y altura 4 es", hipotenusa)

File: cap13/java/Funciones1.java
public class Funciones1 {
	private static int cuadrado(int n) {
		return n * n;
	}
	
	protected static double hipotenusa(int a, int b) {
		return Math.sqrt(cuadrado(a) + cuadrado(b));
	}
	
	public static void main(String[] args) {
		double hipotenusa = hipotenusa(3, 4);
		System.out.println("La hipotenusa de un triángulo de base 3 y altura 4 es "
				+ hipotenusa);
	}
}
